Example/Tests/GINIMultipageReviewControllerTests.swift

```diff
@@ -141,13 +141,46 @@ final class GINIMultipageReviewControllerTests: XCTestCase {
                        "Fifth toolbar item should be the deleteButton")
     }
     
-    func testToolBarAndPagesCollectionContainerColors() {
-        XCTAssertEqual(multipageReviewViewController.toolBar.barTintColor, multipageReviewViewController.pagesCollectionContainer.backgroundColor,
-                       "toolbar and pages collection container background colors should match")
+    func testToolBarTintColor() {
```

Comment:
Didn't we drop the `GINI` file prefix? Why is it still in this one?

Reply:
Not yet, my idea is to do it once we merge `multipage` into develop.